extensions/github-pull-requests/src/PullRequestItem.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+	preferences,
+	List,
+	Icon,
+	ActionPanel,
+	OpenInBrowserAction,
+	CopyToClipboardAction,
+} from "@raycast/api"
+import { PullRequest } from "./types"
+
+export default function PullRequestItem(pullRequest: PullRequest) {
+	const username = preferences.username.value ?? ""
+	const author = pullRequest.author?.login ?? ""
+	const assignees = pullRequest.assignees?.nodes?.map((a) => a.login) ?? []
+	const reviewers =
+		pullRequest.reviewRequests?.nodes?.map((r) => r.requestedReviewer.login) ??
+		[]
+
+	let subtitle = ""
+
+	if (assignees.length > 0) {
+		subtitle += `A: ${assignees.join(", ")}`
+	}
+
+	if (reviewers.length > 0) {
+		if (subtitle.length > 0) {
+			subtitle += " | "
+		}
+
+		subtitle += `R: ${reviewers.join(", ")}`
+	}
+
+	return (
+		<List.Item
+			id={pullRequest.url}
+			key={pullRequest.url}
+			title={pullRequest.title}
+			subtitle={subtitle}
+			accessoryTitle={author === username ? "" : author}
```

Comment:
Is it maybe better to flip those?

Reply:
Maybe also add some last updated date?

Reply:
Flip which those?

Adding last updated date will clutter everything. The idea is good, I'm just not sure on the available space 🤔

Reply:
You could format it to be short. I think @tonka3000 has some good formatting in Twitter and GitLab.